audb/core/api.py

```diff
@@ -426,6 +426,9 @@ def latest_version(
     Returns:
         version string
 
+    Raises:
+        RuntimeError: if no version exists for the requested database
```

Comment:
If we start to list errors here, should we also add the error that no database with the name is found?

Reply:
That's the error I describe here. "no version exists for the requested database" is identical with "no database with the name is found". I only phrased it that way as the error message is showing a related text:

```python
>>> audb.latest_version('emodb2')
...
RuntimeError: Cannot find a version for database 'emodb2'.
```